checkov/common/output/report.py

```diff
@@ -241,6 +241,12 @@ def get_sarif_json(self, tool: str) -> Dict[str, Any]:
         for record in self.failed_checks + self.skipped_checks:
             if self.check_type == CheckType.SCA_PACKAGE and record.check_name != SCA_PACKAGE_SCAN_CHECK_NAME:
                 continue
+
+            help_uri = record.guideline
+            if record.vulnerability_details:
+                # use the CVE link, if it is a SCA record
+                help_uri = record.vulnerability_details.get("link")
```

Comment:
Is this guaranteed to not end up with the `help_uri` being empty? That is forbidden by the same spec.

Reply:
@ankon That's a good catch, thanks! Would you like to contribute a short PR to fix this case? 😄
@gruebel FYI

Reply:
Tried, but my python is rusty: https://github.com/bridgecrewio/checkov/pull/3542